Classes/common/CDTFetchChanges.m

```diff
@@ -91,7 +100,10 @@ - (SequenceNumber)notifyChanges:(TD_RevisionList *)changes
     }
```

Comment:
My concern about the loop was that the number of revisions could be really high so looping through all of them would take a while but if that ever happens, the replication will also take ages and that is a totally different problem.